### PR TITLE
Add Together Java's TJ-Bot

### PR DESCRIPTION
**Pull request by marko-radosavljevic**
Hello everyone,

TJ-Bot is a Discord Bot used on the Together Java server. It is maintained by the community, anyone can contribute!

The idea behind this project is to provide learning experience on a real-world project. Contributors will be provided with our guidance at every step of the way, and friendly but thorough reviews!

Looking forward to your feedback! :relaxed:

### CODE REVIEW

File: _data/projects/TJ-Bot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+desc: TJ-Bot is a Discord Bot used on the Together Java server.
+site: https://github.com/Together-Java/TJ-Bot
+tags:
+  - java
+  - bot
+  - discord
+  - beginner-friendly
+upforgrabs:
+  name: help wanted
+  link: https://github.com/Together-Java/TJ-Bot/labels/help%20wanted
```
Comment (ritwik12) [inline _data/projects/TJ-Bot.yml:11]: @marko-radosavljevic Please use this instead https://github.com/Together-Java/TJ-Bot/labels/help%20wanted

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/TJ-Bot.yml (hunk 1 above)
Comment (marko-radosavljevic) [reply _data/projects/TJ-Bot.yml:11]: @ritwik12 , thanks for the review ^^

Currently only have phone available, but it seems already correct to me.

@shiftkey mentioned in other PRs that the bot likely fails due to a bug:
https://github.com/up-for-grabs/up-for-grabs.net/pull/3330#issuecomment-1228885093

Comment (ritwik12) [reply _data/projects/TJ-Bot.yml:11]: Got it, seems to be that only.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@marko-radosavljevic Thanks!!

**Pull request merged**